src/panels/config/cloud/register/cloud-register.ts

```diff
@@ -229,13 +229,13 @@ export class CloudRegister extends LitElement {
   private async _handleResendVerifyEmail() {
     const emailField = this._emailField;
 
-    const email = emailField.value;
-
     if (!emailField.reportValidity()) {
       emailField.focus();
       return;
     }
 
+    const email = emailField.value.toLowerCase();
```

Comment:
Does this mean that if someone registered before this PR with a upper case char he would no longer be able to resend the verify email? (probably small chance)

Reply:
Yup, will implement the same try logic here (and for forgot password as well)